libcobblersignatures/__init__.py

```diff
@@ -50,7 +50,7 @@ def signaturesjson(self, value: str):
         :param value: The str which will be loaded by the Python json utility.
         """
         if value is None:
-            pass
+            return
```

Comment:
This is really a nitpick, I prefer to be explicit about what is being returned. But no need to change it if you prefer an implict `None`